dozent/__main__.py

```diff
@@ -72,7 +72,7 @@
 
         if command_line_arguments["timeit"]:
             print(
-                f"Download Time: {datetime.timedelta(seconds=(time.time() - _start_time))}"
+                f"\nDownload Time: {datetime.timedelta(seconds=(time.time() - _start_time))}"
```

Comment:
Any specific reason why you're using `datetime.timedelta` instead of just subtracting them? I guess better formatting right?

Reply:
This was a while ago and I'm not sure if that was me or Keelimeguy - but I have no issue with the formatting.